crates/services/gas_price_service/src/v1/metadata.rs

```diff
@@ -92,7 +92,9 @@ impl From<&V1AlgorithmConfig> for AlgorithmUpdaterV1 {
             .map(|size| u128::from(*size))
             .sum();
         Self {
-            new_scaled_exec_price: value.new_exec_gas_price,
+            new_scaled_exec_price: value
```

Comment:
The `gas_price_factor` is just a scaling factor to make sure we can track the change between small values. Like we do for gas costs. If the gas price is 1, it is stored as 100 as the `scaled` value, then we can increase it by 10% or 5% under the hood but it will still be returned as 1 until it crosses 150 or whatever.

Reply:
The gas price factor can be different on different heights. Do we take it into account? 

I though we work with Wei and because of that we don't need to scale gas price.

Reply:
What if the price is 1 Wei, which it often is?

Reply:
I'll add a test.

Reply:
I added a TODO to remove this with this issue:
https://github.com/FuelLabs/fuel-core/issues/2481

Reply:
It would be better if you added todo for the field of the config instead of here=)

Reply:
I'll move it in my next pr. Gonna merge.

Reply:
Moved here:
https://github.com/FuelLabs/fuel-core/pull/2468/commits/abf0cca485b22cfcad204d98d5c2ebcb2f47934a